### DISCUSSION TITLE
关于webui 规划的疑问

### DISCUSSION THREAD

**Discussion by shade233** [lifegpc/bili#71] [Q&A]
webui未来打算做成一种怎样的模式？

1. 是否会像下载软件那样，在远端输入AV/BV号就自动在电脑上下载
2. 还是做成在webui上输入AV/BV号然后出现下载链接，方便访问webui的设备下载（比如手机）？

目前貌似没有下载完整视频的功能，是ffmpeg的原因吗？

1. ffmpeg貌似有网页上可用的版本，比如ffmpeg Webassembly
2. 或者在运行webui的电脑上合成后提供下载。然而不可能让电脑先下载再合成最后删掉吧，这样极大浪费带宽。

webui打算运行在类似服务器的场景还是工作站的场景？

看到webui提示有些设置要重启webui，webui是否能在远端直接重启？（比如一个Button）

**Comment by lifegpc** [comment]
第一个问题，主要是考虑1，但也提供了直接可以访问的链接复制（有反代来解决需要referer的问题）
第二个是因为还没开发完。
WEBUI我的打算就是提供一个UI，所以说我考虑的是在本地（localhost）运行的

**Comment by lifegpc** [comment]
需要重启webui是因为目前在更新设置后，程序里现有已读取的更新不会被更新。
